fgpyo/alignment/__init__.py

```diff
@@ -0,0 +1,232 @@
+from typing import Callable
+from typing import List
+
+from fgpyo.sam import Cigar
+from fgpyo.sam import CigarElement
+from fgpyo.sam import CigarOp
+
+
+class Alignment:
```

Comment:
Any reason not to make this a frozen (i.e. immutable) attr class? See: https://github.com/fulcrumgenomics/fgpyo/blob/5ef0f5e9ced1e8e3763e6858317b59e448193635/fgpyo/sam/__init__.py#L511

For container classes, we should use attrs, dataclasses, or named tuples I think.